Tasks/AzureCLIV2/typings.json

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I think the typings are no longer required. You can use @types directly in package.json